packages/ui/src/components/molecules/CheckboxList.tsx

```diff
@@ -2,7 +2,11 @@ import React, { Dispatch, FC, SetStateAction } from 'react';
 
 import { CheckBox, Flex } from '../atoms';
 
-export type CheckboxListItems = { label: string; checked: boolean }[];
+export type CheckboxListItems = {
+  label: string;
+  checked: boolean;
+  __id: string | undefined;
```

Comment:
Can we just change it to `id`?
```suggestion
 id: string | undefined;
```